First round of README tweaks (#33)
paritytech/ledgeracio: README.md

# WARNING: This is alpha quality software and not suitable for production. It is incomplete and will have bugs.

# Ledgeracio CLI

Ledgeracio is a command-line tool and Ledger app designed for staking operations
on Substrate-based networks.

Running `ledgeracio --help` will provide top-level usage instructions.

Ledgeracio CLI is intended to work with a special Ledgeracio Ledger app, but
most of its commands will work with stock Kusama or Polkadot Ledger apps as well.
This is less secure, however, as these apps do not enforce the same restrictions
that the Ledgeracio app does.  Using a stock app in production is not
recommended.

The Polkadot app can be found [here](https://github.com/zondax/ledger-polkadot)
and the Kusama app can be found [here](https://github.com/zondax/ledger-kusama).
Other Substrate-based chains are currently not supported, but local devnets

> should work as long as their RPC endpoints match Kusama/Polkadot's.

Reviewer: ```suggestion
should work as long as their RPC API matches Kusama/Polkadot's.
```

Ledgeracio must work with any RPC endpoint URL, so long as it is wire-compatible.

Ledgeracio only support Unix-like systems, and has only tested mostly on Linux.

> That said, it works on macOS and other Unix-like systems that provide the

Reviewer: Have you tested it on macOS?

Author: I have yes.

system administrator, and they cannot be written to except by the administrator.
This is to prevent accidental overwrites.

The public key is not sensitive, and is required by anyone who wishes to verify

> signed allowlists and operate on the whitelisted accounts.  It will be uploaded

Reviewer: ```suggestion
signed allowlists and operate on the allowed accounts.  It will be uploaded
```

Ledgeracio does not use the term “whitelist”.

Author: Good call and good riddance. :)

### Metadata inspection: `ledgeracio metadata`

This command takes no arguments.  It pretty-prints the chain metadata to stdout.

> It is primarily intended for debugging. Requires a network connection.

Reviewer: ```suggestion
It is primarily intended for debugging.  Requires a network connection.
```

Extremely minor style nit: Ledgeracio uses two spaces after a period.

Author: I noticed that! Hopefully one day we can sit down and debate that choice properly! ;)

I'll stick to the 2x:space: rule from here on out, but yeah, I do it under protest!

### Properties inspection: `ledgeracio properties`

This command takes no arguments.  It pretty-prints the chain properties to

> stdout.  It is primarily intended for debugging. Requires a network connection.

Reviewer: ```suggestion
stdout.  It is primarily intended for debugging.  Requires a network connection.
```

Ditto

### Nominator operations: `ledgeracio nominator`

This command performs operations using nominator keys ― that is, keys on a

> nominator derivation path. Requires a network connection.  The following

Reviewer: ```suggestion
nominator derivation path.  Requires a network connection.  The following
```

Ditto

This command takes an index as argument, and sets the payment target.  The
target must be one of `Stash`, `Staked`, or `Controller` (case-insensitive).

### Validator operations: `ledgeracio validator`

> This command handles validator operations. Requires a network connection.  It

Reviewer: ```suggestion
This subcommand handles validator operations.  It requires a network connection, and
```
